### ISSUE TITLE
Executable permissions issue on Eddie cluster affecting riboviz repo

### ISSUE THREAD

**Issue by FlicAnderson** [riboviz/riboviz#444]
New issue created from permissions / git issue arising in Eddie compute cluster: 

The problem [initially described here](https://github.com/riboviz/example-datasets/pull/64#issuecomment-936352496) 

> Logging in & moving to my riboviz-flic folder on the wallace lab drive, I've encountered a set of files labelled as 'modified' when I run git status. This is AFTER deleting and re-cloning my example-datasets repo folder into my riboviz-flic folder.
```
(riboviz) (base) [fanders6@login02(eddie) example-datasets]$ git status
# On branch main
# Changes not staged for commit:
#   (use "git add <file>..." to update what will be committed)
#   (use "git checkout -- <file>..." to discard changes in working directory)
#
#	modified:   fungi/candida/annotation/Candida_albicans_CDS_with_120utrs.fa
#	modified:   fungi/candida/annotation/Candida_albicans_CDS_with_120utrs.gff3
#	modified:   fungi/candida/contaminants/Candida_albicans_rRNA_tRNA.fa
#	modified:   fungi/saccharomyces/annotation/Saccharomyces_cerevisiae_yeast_CDS_w_250utrs.fa
#	modified:   fungi/saccharomyces/annotation/Saccharomyces_cerevisiae_yeast_CDS_w_250utrs.gff3
#	modified:   fungi/saccharomyces/contaminants/Saccharomyces_cerevisiae_yeast_rRNA_R64-1-1.fa
#	modified:   simulated/mok/annotation/Scer_YAL_5genes_w_250utrs.fa
#	modified:   simulated/mok/annotation/Scer_YAL_5genes_w_250utrs.gff3
#	modified:   simulated/mok/annotation/tiny_2genes_20utrs.gff3
#	modified:   simulated/mok/contaminants/Sc_rRNA_example.fa
#
no changes added to commit (use "git add" and/or "git commit -a") 
```
> When I checked git diff to investigate what the changes were, I received this:
```
(riboviz) (base) [fanders6@login02(eddie) example-datasets]$ git diff fungi/candida/annotation/Candida_albicans_CDS_with_120utrs.fa
diff --git a/fungi/candida/annotation/Candida_albicans_CDS_with_120utrs.fa b/fun
old mode 100755
new mode 100644 
``` 

Further checking of permissions settings within the riboviz shared group folder: 

Showing here the current ls -l contents of the group folder (`wallace_rna`), and then within my `riboviz-flic/riboviz/` folder within the group folder: 

```
(base) [fanders6@login02(eddie) ~]$ cd /exports/csce/eddie/biology/groups/wallace_rna/

(base) [fanders6@login02(eddie) wallace_rna]$ ls -al
total 672
drwx--S--- 28 root     datastore_biology_groups_wallace_rna 2048 Jul 16 19:48 .
drwxr-xr-x 53 root     root                                 4096 Sep 10 11:39 ..
drwx--S---  3 fanders6 datastore_biology_groups_wallace_rna  512 Sep 27  2020 20200925_W-Cn-H99_2020
drwx--S---  3 fanders6 datastore_biology_groups_wallace_rna  512 Sep 28  2020 20200928_W-Cn-JEC21_2020
# ... bunch of folders with similar permissions ...
drwx--S---  6 fanders6 datastore_biology_groups_wallace_rna  512 May  4 16:37 20210422_K-Sc-eEF3_2019
drwx--S---  4  2233078 datastore_biology_groups_wallace_rna  512 Jun  3  2020 anaconda
drwx--S---  9 ewallac2 datastore_biology_groups_wallace_rna  512 Nov 20  2020 CryptoRNASeq2020_draft
drwx--S---  2 ewallac2 datastore_biology_groups_wallace_rna  512 Nov 18  2020 EW_test
drwx--S---  5 fanders6 datastore_biology_groups_wallace_rna  512 Jun 16 18:09 fastq-datafiles
drwx--S---  3  2233078 datastore_biology_groups_wallace_rna  512 Oct  2  2020 .nextflow
-rw-------  1 fanders6 datastore_biology_groups_wallace_rna 6553 Oct  2  2020 .nextflow.log
-rw-------  1  2233078 datastore_biology_groups_wallace_rna 6449 Jun  4  2020 .nextflow.log.1
-rw-------  1 fanders6 datastore_biology_groups_wallace_rna  257 Oct 17  2020 nxfl-vig-symlk-7451483-node3g17.ecdf.ed.ac.uk.e
-rw-------  1 fanders6 datastore_biology_groups_wallace_rna 1926 Oct 17  2020 nxfl-vig-symlk-7451483-node3g17.ecdf.ed.ac.uk.o
drwx--S---  3  2233078 datastore_biology_groups_wallace_rna  512 Jun  4  2020 .pytest_cache
-rw-------  1 root     datastore_biology_groups_wallace_rna    0 Oct 15  2020 .quotaupdated
-rw-------  1 fanders6 datastore_biology_groups_wallace_rna 4652 Oct  2  2020 .RData
drwx--S---  5  2233078 datastore_biology_groups_wallace_rna  512 Jun  4  2020 regression-test-data-2.0.beta
-rw-------  1 fanders6 datastore_biology_groups_wallace_rna 3276 Oct  2  2020 .Rhistory
drwx--S--- 16  2233078 datastore_biology_groups_wallace_rna 4096 Sep 14  2020 riboviz
drwx--S---  3 s1734289 datastore_biology_groups_wallace_rna  512 Jun 21 10:24 riboviz-emma
drwx--S---  3 fanders6 datastore_biology_groups_wallace_rna  512 Sep  9 15:55 riboviz-flic
drwx--S---  3 s2039234 datastore_biology_groups_wallace_rna  512 Aug 30 10:52 riboviz-jcliu
drwx--S--- 15 mjj      datastore_biology_groups_wallace_rna 2048 Jun 10  2020 riboviz-mikej
drwx--S---  3 s1767247 datastore_biology_groups_wallace_rna  512 Jun 21 10:47 riboviz-soph
drwx--S--- 69 fanders6 datastore_biology_groups_wallace_rna 4096 Mar 24  2021 Rlibrary
drwxr-xr-x  6  2233078 eddie_users                           512 Jun 21  2020 Wallace_2020_H99
drwxr-xr-x  4  2233078 eddie_users                           512 Jun 21  2020 Wallace_2020_JEC21
drwx--S---  6  2233078 datastore_biology_groups_wallace_rna  512 Oct  2  2020 work


(base) [fanders6@login02(eddie) wallace_rna]$ ls -al riboviz-flic/riboviz/
total 8
drwx--S---  4 fanders6 datastore_biology_groups_wallace_rna  512 Oct  6 15:13 .
drwx--S---  3 fanders6 datastore_biology_groups_wallace_rna  512 Sep  9 15:55 ..
drwx--S--- 11 fanders6 datastore_biology_groups_wallace_rna 2048 Oct  6 15:13 example-datasets
drwx--S--- 12 fanders6 datastore_biology_groups_wallace_rna 2048 Sep 15 15:53 riboviz
-rw-------  1 fanders6 datastore_biology_groups_wallace_rna  309 Sep  9 16:00 set-riboviz-env.sh
``` 

This issue prevented working with riboviz as the majority of files were listed as having modified permissions, which wasn't helpful for version control.

**Comment by ewallace** [comment]
Closed by #445

**Comment by FlicAnderson** [comment]
Moving to 'done' as it's merged and closed.
